lib/matplotlib/dviread.py

```diff
@@ -990,6 +990,8 @@ def find_tex_file(filename, format=None):
     kpathsea. It is also available as part of MikTeX, a popular
     distribution on Windows.
 
+    *If the file is not found, an empty string is returned*.
```

Comment:
As it turns out, this less-than-optimal behavior comes from not checking the returncode of the Popen in the old code, and calling code expects that...

Reply:
Are you sure that it will be fine?

`PsFont` will contain an empty path to font
https://github.com/matplotlib/matplotlib/blob/01d65f3f945e24410e65ae1b52e1cbc58d08df0e/lib/matplotlib/dviread.py#L844-L848

`Encoding` tries to `open` the path and will fail
https://github.com/matplotlib/matplotlib/blob/01d65f3f945e24410e65ae1b52e1cbc58d08df0e/lib/matplotlib/textpath.py#L19-L20
https://github.com/matplotlib/matplotlib/blob/01d65f3f945e24410e65ae1b52e1cbc58d08df0e/lib/matplotlib/dviread.py#L956-L959

`PsfontsMap` also tries to `open` the path and will fail
https://github.com/matplotlib/matplotlib/blob/c07d6b84adaebed17283a051050b002af881b6f7/lib/matplotlib/backends/backend_pdf.py#L661
https://github.com/matplotlib/matplotlib/blob/c07d6b84adaebed17283a051050b002af881b6f7/lib/matplotlib/backends/backend_pdf.py#L674
https://github.com/matplotlib/matplotlib/blob/01d65f3f945e24410e65ae1b52e1cbc58d08df0e/lib/matplotlib/dviread.py#L821-L823